docs/troubleshooting.md

```diff
@@ -0,0 +1,7 @@
+# Troubleshooting
+
+## If the load balancer for edge never becomes healthy target at startup
```

Comment:
This section is a bit thin. Also, a point of style. We might shorten the header to something like

```
## Load Balancer Targets Unhealthy

If the load balancer never detects healthy targets for edge...
```

Reply:
Perhaps also add a hyperlink to cloudwatch docs.